scrypto-test/src/environment/internal.rs

```diff
@@ -4,130 +4,66 @@
 use super::*;
 use crate::prelude::*;
 
+// TODO: I would like to remove the reliance on `CommittableSubstateDatabase` and to instead commit
+//       everything to the track. As in, nothing ever gets committed to the database. Even the
+//       initial bootstrapping should be done in this way. This mainly comes from a desire to use
+//       the node's database with scrypto-test, and it does not implement that trait.
```

Comment:
This is actually a highly desired feature. From my interaction with Hacken, they wanted to run test transactions on the Node's database.